CMake/fletch-tarballs.cmake

```diff
@@ -519,6 +519,12 @@ if(NOT WIN32)
   list(APPEND fletch_external_sources Caffe_Segnet)
 endif()
 
+# LibSVM
+set(Libsvm_url "https://github.com/m-chaturvedi/libsvm-3.1-custom/archive/master.zip")
```

Comment:
I was testing it using my own repository.  I will check if I have permissions to upload it where we want it, https://gitlab.kitware.com/kwiver/

Reply:
That isn't exactly what I meant. The question is, why are we using a fork of https://www.csie.ntu.edu.tw/~cjlin/libsvm/ instead of using the upstream library itself? Have we added features that we can't push upstream? Please explain the need to fork it rather than use it.